app/controllers/candidate_interface/referees_controller.rb

```diff
@@ -115,8 +125,8 @@ def referee_params
         .transform_values(&:strip)
     end
 
-    def redirect_to_review_referees_if_amendable
-      redirect_to candidate_interface_review_referees_path if current_application.amendable?
+    def redirect_to_review_referees_if_amendable_and_two_referees_are_present
+      redirect_to candidate_interface_review_referees_path if current_application.amendable? && current_application.application_references.count == ApplicationForm::MINIMUM_COMPLETE_REFERENCES
     end
```

Comment:
I don't think this change is necessary because it won't do anything, `amendable?` is always false because the `edit_application` feature flag is permanently disabled.